app/screens/home-screen/home-screen.tsx

```diff
@@ -137,6 +150,16 @@ export const HomeScreen: React.FC = () => {
 
   const onMenuClick = (target: Target) => {
     if (isAuthed) {
+      if (
+        target === "receiveBitcoin" &&
+        !hasPromptedSetDefaultAccount &&
+        dataAuthed?.me?.defaultAccount?.transactions?.edges?.length ===
```

Comment:
```suggestion
        dataAuthed?.me?.defaultAccount?.transactions?.edges?.length >=
```

Reply:
should probably `more or equal` than `equal`. imagine you do the quiz, you will get multiple tx without going back to the home screen

Reply:
I have it set equal to `===` so that if a person logs out and logs back in, they shouldnt get asked again.

Reply:
fair assumption. not sure how we should deal with the earn section.

Reply:
Interesting, I didn't think of that scenario. I suppose it could be better to show it multiple times rather that have some people be skipped. Ultimately, I think we need a Apollo Federation with a separate api that keeps track of things like this.

Reply:
Although if we switch it to be `>==` it will also show up for every existing account. Which maybe could be good.